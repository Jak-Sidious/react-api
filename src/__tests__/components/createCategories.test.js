import React from 'react';
import Enzyme, { mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { Form, Button } from 'semantic-ui-react';
import { notify } from 'react-notify-toast';
import { MemoryRouter } from 'react-router-dom';

import CreateCategory from '../../components/categories/createCategory';

Enzyme.configure({ adapter: new Adapter() });

describe('<CreateCategory />', () => {
  const pathname = 'path';
  const props = {
    location: { pathname }
  }

  it('should render without crashing', () => {
    const { enzymeWrapper } = mount(
      <MemoryRouter>
        <CreateCategory {...props}/>
      </MemoryRouter>
    );
  });

});
