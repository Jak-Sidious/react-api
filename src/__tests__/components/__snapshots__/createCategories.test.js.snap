// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CreateCategory /> should render properly 1`] = `
<div>
  <Navigation />
  <_class
    options={Object {}}
    pathname="path"
  />
  <div
    className="createCat"
  >
    <div
      className="wrapper"
    >
      <Form
        as="form"
        className="form-signin"
        onSubmit={[Function]}
      >
        <FormInput
          as={[Function]}
          autoFocus={true}
          className="catNameField"
          control={[Function]}
          name="category_name"
          onChange={[Function]}
          placeholder="Enter your category Name"
          type="category_name"
        />
        <FormTextArea
          as={[Function]}
          className="catDescField"
          control={[Function]}
          name="category_description"
          onChange={[Function]}
          placeholder="Enter the Description"
          type="category_description"
        />
        <Button
          as="button"
          className="left floated"
          color="blue"
          fluid={true}
          id="submitButton"
          size="large"
          type="submit"
          value="submit"
        >
          Create Category
        </Button>
        <br />
      </Form>
    </div>
  </div>
</div>
`;
